### PR TITLE
Correct some translation errors.

### PR DESCRIPTION
**Pull request by zhuyuanxiang**
Easy to read and understand.
Complete some equation description.

### CODE REVIEW

**Comment by mli** [comment]
Job d2l-zh/PR-717/1 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-717/

**Comment by zhuyuanxiang** [comment]
Thks for your hard word.

**Comment by mli** [comment]
Job d2l-zh/PR-717/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-717/

File: chapter_attention-mechanisms/attention-scoring-functions.md [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
+在 :numref:`sec_nadaraya-waston` 中，我们使用高斯核来模拟查询和键之间的交互。将 :eqref:`eq_nadaraya-waston-gaussian` 中的高斯核的指数视为 * 注意力评分函数 *（简称 * 评分函数 *），这个函数的结果基本上被输入了 softmax 操作。因此，我们获得了与键配对的值的概率分布（注意力权重）。最后，注意力集中的输出只是基于这些注意力权重的值的加权总和。
 
-从较高层面来说，我们可以使用上述算法实例化 :numref:`fig_qkv` 中的注意机制框架。:numref:`fig_attention_output` 表示 $a$ 的注意力评分函数，说明了如何将注意力集中的输出计算为加权值和。由于注意力权重是概率分布，因此加权总和基本上是加权平均值。
+从较高层面来说，我们可以使用上述算法实例化 :numref:`fig_qkv` 中的注意力机制框架。:numref:`fig_attention_output` 表示 $a$ 的注意力评分函数，说明了如何将注意力池化的输出计算为加权值和。由于注意力权重是概率分布，因此加权和本质上是加权平均值。
 
 ![Computing the output of attention pooling as a weighted average of values.](../img/attention-output.svg)
 :label:`fig_attention_output`
 
-从数学上讲，假设我们有一个查询 $\mathbf{q} \in \mathbb{R}^q$ 和 $m$ 键值对 $(\mathbf{k}_1, \mathbf{v}_1), \ldots, (\mathbf{k}_m, \mathbf{v}_m)$，其中任何 $\mathbf{k}_i \in \mathbb{R}^k$ 和任何 $\mathbf{v}_i \in \mathbb{R}^v$。注意力池 $f$ 被实例化为值的加权总和：
+从数学上讲，假设我们有一个查询 $\mathbf{q} \in \mathbb{R}^q$ 和 $m$ 个键-值对 $(\mathbf{k}_1, \mathbf{v}_1), \ldots, (\mathbf{k}_m, \mathbf{v}_m)$，其中任何 $\mathbf{k}_i \in \mathbb{R}^k$ 和任何 $\mathbf{v}_i \in \mathbb{R}^v$。注意力池化函数 $f$ 被实例化为值的加权总和：
```
Comment (goldmermaid) [inline chapter_attention-mechanisms/attention-scoring-functions.md:12]: Great catch! Can you add a quotation mark (i.e. change to "键-值"对)?

File: chapter_attention-mechanisms/attention-scoring-functions.md [hunk 2]
```diff
@@ -202,26 +204,26 @@ attention.eval()
...
-## 缩放点-产品关注
+## 缩放的点积注意力
 
-计分功能的计算效率更高的设计可以简单地是点积。但是，点积操作要求查询和键具有相同的矢量长度，比如 $d$。假设查询的所有元素和关键字都是独立的随机变量，均值和单位方差零。两个向量的点积均值为零，方差为 $d$。为确保无论矢量长度如何，点积的方差仍然是一个，* 缩放的点积注意 * 评分功能
+设计具有更高计算效率的评分函数可以使用点积。但是，点积操作要求查询和键具有相同的矢量长度。假设查询和键的所有元素都是独立的随机变量，并且都是零均值和单位方差。两个向量的点积均值为零，方差为 $d$。为确保无论矢量长度如何，点积的方差在不考虑向量长度的情况下仍然是 $1$，则 * 缩放的点积注意力 * 评分函数
 
 $$a(\mathbf q, \mathbf k) = \mathbf{q}^\top \mathbf{k}  /\sqrt{d}$$
 
-将点积除以 $\sqrt{d}$。在实践中，我们通常以微型批量来考虑提高效率，例如 $n$ 查询和 $m$ 键值对的计算注意力，其中查询和键的长度为 $d$，值的长度为 $v$。查询 $\mathbf Q\in\mathbb R^{n\times d}$、键 $\mathbf K\in\mathbb R^{m\times d}$ 和值 $\mathbf V\in\mathbb R^{m\times v}$ 的扩展点-产品关注度是
+将点积除以 $\sqrt{d}$。在实践中，我们通常以小批量来考虑提高效率，例如 $n$ 个查询和 $m$ 个键-值对下计算注意力，其中查询和键的长度为 $d$，值的长度为 $v$。查询 $\mathbf Q\in\mathbb R^{n\times d}$、键 $\mathbf K\in\mathbb R^{m\times d}$ 和值 $\mathbf V\in\mathbb R^{m\times v}$ 的缩放的点积注意力是
```
Comment (goldmermaid) [inline chapter_attention-mechanisms/attention-scoring-functions.md:221]: same here

File: chapter_attention-mechanisms/nadaraya-waston.md [hunk 3]
```diff
@@ -20,13 +21,13 @@ from torch import nn
 
 ## 生成数据集
 
-为了简单起见，让我们考虑以下回归问题：给定输入-产出对 $\{(x_1, y_1), \ldots, (x_n, y_n)\}$ 的数据集，如何学习 $f$ 来预测任何新输入 $\hat{y} = f(x)$ 的输出 $\hat{y} = f(x)$？
+简单起见，考虑回归问题：给定输入-输出对 $\{(x_1, y_1), \ldots, (x_n, y_n)\}$ 的数据集，如何学习 $f$ 来预测任何新输入 $x$ 的输出 $\hat{y} = f(x)$？
```
Comment (goldmermaid) [inline chapter_attention-mechanisms/nadaraya-waston.md:24]: same here: "输入-输出"对

**Review by goldmermaid — COMMENTED** [review]
Thanks for your contribution @zhuyuanxiang !

**Comment by mli** [comment]
Job d2l-zh/PR-717/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-717/

**Comment by zhuyuanxiang** [comment]
> Thanks for your contribution @zhuyuanxiang !

I have fixed the errors you have mentioned.

**Review by goldmermaid — APPROVED** [review]
LGTM!

**Comment by goldmermaid** [comment]
Amazing work @zhuyuanxiang ! Thanks!

**Pull request merged**